libp2p/network/stream/exceptions.py

```diff
@@ -0,0 +1,17 @@
+from libp2p.exceptions import BaseLibp2pError
+
+
+class StreamError(BaseLibp2pError):
+    pass
+
+
+class StreamEOF(StreamError, EOFError):
+    pass
+
+
+class StreamReset(StreamError):
+    pass
+
+
+class StreamClosed(StreamError):
+    pass
```

Comment:
nice error tree :)

Reply:
Thanks a lot! Trying to make it easier to handle.